cmd/aro/create.go

```diff
@@ -167,7 +168,7 @@ func _makeInstaller(ctx context.Context, log *logrus.Entry) (installer.Interface
 	if err != nil {
 		return nil, err
 	}
-	fpAuthorizer, err := _env.FPAuthorizer(sub.Properties.TenantID, _env.Environment().ResourceManagerEndpoint)
+	fpAuthorizer, err := refreshable.NewAuthorizer(_env, sub.Properties.TenantID)
```

Comment:
I thought we were going to revert this change?  (cc: @SudoBrendan)

Reply:
I think this is OK to leave.  Originally `FPAuthorizer` returned a `refreshable` now we call `refreshable.NewAuthorizer` directly which uses `FPAuthorizer` underneath.  When trying to revert this it opened a can of worms since the new version of `FPAuthorizer` returns an `autorest.Authorizer` instead of a `refreshable.Authorizer`.

Reply:
Yes indeed when we reverted it back many things were breaking.